.github/workflows/ci.yml

```diff
@@ -29,11 +29,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        os:
+          - ubuntu-latest
+          - windows-latest
         shard_file:
           - shard.yml
         postgres_version:
           - 14
-          - 15
```

Comment:
No need to check every single version. I think just checking the lowest and the highest versions is fine and assume everything in the middle "just works".